### PR TITLE
chore(example): correct the view source link

### PR DESCRIPTION
**Pull request by zarov**

### CODE REVIEW

File: examples/index.html [hunk 1]
```diff
@@ -51,10 +51,10 @@
         iframe.focus();
 
         var viewSrc = '<a href="https://github.com/iTowns/itowns/blob/master/examples/';
-        viewSrc += window.location.hash.substr(1);
+        viewSrc += window.location.hash.substr(1) || 'view_3d_map';
```
Comment (gchoqueux) [inline examples/index.html:54]: The `code source` history is fixed?

**Comment by gchoqueux** [comment]
I also have two comments on the examples (which can be fixed later):

- It's possible to loose the focus, so the `ctrl` key doesn't handled.
- Could you export `view` instance to debug iTowns in console?

File: examples/index.html (hunk 1 above)
Comment (zarov) [reply examples/index.html:54]: Unfortunately I couldn't make it work. Three.js, which has inspired this template, haven't find a solution too. It doesn't seems possible from what I've searched.

**Comment by zarov** [comment]
> Could you export view instance to debug iTowns in console?

If you want to debug, you can still open the examples outside this view, by remove the `#` from the url and adding `.html` at its end.

> It's possible to loose the focus, so the ctrl key doesn't handled.

I can't find a solution for the moment unfortunately.

**Review by gchoqueux — APPROVED** [review]

**Pull request merged**